geopolars/src/geoseries.rs

```diff
@@ -30,8 +30,52 @@ pub enum TransformOrigin {
     Point(Point),
 }
 
+pub enum BroadcastableParameter<'a, T> {
+    Val(T),
+    Series(&'a Series),
+}
+
+impl<'a> From<f64> for BroadcastableParameter<'a, f64> {
+    fn from(val: f64) -> BroadcastableParameter<'a, f64> {
+        BroadcastableParameter::Val(val)
+    }
+}
+
+impl<'a> From<i64> for BroadcastableParameter<'a, i64> {
+    fn from(val: i64) -> BroadcastableParameter<'a, i64> {
+        BroadcastableParameter::Val(val)
+    }
+}
+
+impl<'a> From<u64> for BroadcastableParameter<'a, u64> {
+    fn from(val: u64) -> BroadcastableParameter<'a, u64> {
+        BroadcastableParameter::Val(val)
+    }
+}
+
+impl<'a, T> From<&'a Series> for BroadcastableParameter<'a, T> {
+    fn from(val: &'a Series) -> BroadcastableParameter<T> {
+        BroadcastableParameter::Series(val)
+    }
+}
+
+pub fn test_fn<'a>(
+    x: impl Into<BroadcastableParameter<'a, i64>>,
+    y: impl Into<BroadcastableParameter<'a, f32>>,
```

Comment:
Maybe another dumb question but how does this work with `BroadcastableParameter<'a, f32>`, when you only defined `From` on `f64`?

Reply:
Oh I hadn't noticed that myself, I think I had probably meant to put f64 there. Presumably it works because f32 can be coerced in to an f64? If we had defined it the other way around, implemented the From for f32 and try a `impl Into<BroadcastableParameter<'a,f64>` it would have failed